app/views/posts/index.html.erb

```diff
@@ -1 +1,8 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<div id="postTemplate">
+   <%= render partial: "shared/single_user", locals: { user: @user } %>
+<% @user.posts.each do |post| %>
+     <%= render partial: "shared/single_post", locals: { post: post } %>   
+     <%= render partial: "shared/collection_comments", locals: { post: post, user: @user } %>    
+<% end %>
```

Comment:
- [ ] It should use `post.recent_comments` method to render 5 most recent comments. At the moment you are rendering all the post's comments. Please kindly ensure you only render the 5 most recent comments.